app/views/arts/index.html.erb

```diff
@@ -1,9 +1,108 @@
-<h1>Welcome to iSpy</h1>
+
+<html>
+<head>
+<title>iSpy</title>
+</head>
+
+
+<body>
+  <br>
```

Comment:
Try to avoid using `<br>` tags at all possible costs! All they do is add space/line breaks, which is something that should (in most situations) be done with CSS.